whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapServiceAclTestIntegration.java

```diff
@@ -120,4 +170,167 @@ public void lookup_mntner_acl_not_counted() throws Exception {
     }
+
+    @Test
+    public void lookup_inetnum_filtered_emails_acl_not_counted() throws Exception {
+        databaseHelper.addObject("" +
+                "inetnum:      192.0.2.0 - 192.0.2.255\n" +
+                "netname:      TEST-NET-NAME\n" +
+                "org:      ORG1\n" +
+                "descr:          Test IPv4\n" +
+                "country:        NL\n" +
+                "tech-c:         TP1-TEST\n" +
+                "admin-c:        ROLE_TEST\n" +
+                "status:         OTHER\n" +
+                "mnt-by:         OWNER-MNT\n" +
+                "created:         2022-08-14T11:48:28Z\n" +
+                "last-modified:   2022-10-25T12:22:39Z\n" +
+                "source:         TEST");
+        ipTreeUpdater.rebuild();
+
+        final Ip entity = createResource("ip/192.0.2.1")
+                .request(MediaType.APPLICATION_JSON_TYPE)
+                .get(Ip.class);
+
+        assertThat(entity.getHandle(), is("192.0.2.0 - 192.0.2.255"));
+        assertThat(entity.getEntitySearchResults().size(), is(4));
+
+        assertThat(entity.getEntitySearchResults().get(0).getHandle(), is("ORG1"));
+        assertThat(entity.getEntitySearchResults().get(0).getVCardArray().toString(), is("" +
+                "[vcard, [[version, {}, text, 4.0], " +
+                "[fn, {}, text, TEST], " +
+                "[kind, {}, text, org]]]"));
+
+        assertThat(entity.getEntitySearchResults().get(1).getHandle(), is("OWNER-MNT"));
+        assertThat(entity.getEntitySearchResults().get(1).getVCardArray().toString(), is("" +
+                "[vcard, [[version, {}, text, 4.0], " +
+                "[fn, {}, text, OWNER-MNT], " +
+                "[kind, {}, text, individual]]]"));
+
+        assertThat(entity.getEntitySearchResults().get(2).getHandle(), is("ROLE_TEST"));
+        assertThat(entity.getEntitySearchResults().get(2).getVCardArray().toString(), is("" +
+                "[vcard, [[version, {}, text, 4.0], " +
+                "[fn, {}, text, Test Role], " +
+                "[kind, {}, text, group], " +
+                "[adr, {label=Singel 258}, text, [, , , , , , ]], " +
+                "[tel, {type=voice}, text, +31 6 12345678]]]"));
+
+        assertThat(entity.getEntitySearchResults().get(3).getHandle(), is("TP1-TEST"));
+        assertThat(entity.getEntitySearchResults().get(3).getVCardArray().toString(), is(""+
+                "[vcard, [[version, {}, text, 4.0], " +
+                "[fn, {}, text, Test Person], [kind, {}, text, individual], " +
+                "[adr, {label=Singel 258}, text, [, , , , , , ]], " +
+                "[tel, {type=voice}, text, +31 6 12345678]]]"));
+
+        assertThat(testPersonalObjectAccounting.getQueriedPersonalObjects(InetAddress.getByName(LOCALHOST)), is(0));
+    }
+
+    @Test
+    public void lookup_inet6num_filtered_emails_acl_not_counted() throws Exception {
```

Comment:
Test that related entities e-mail is redacted

Reply:
Yes, done